src/FirebaseHelper.ts

```diff
@@ -0,0 +1,29 @@
+import firebase from '@firebase/app';
+import '@firebase/storage';
+
+export default class FirebaseHelper {
+  private _storage = firebase.storage!();
+  private _storageRef = this._storage.ref();
+
+  public getDownloadUrl = (file: string) => {
+    try {
+      return this._storageRef.child(file).getDownloadURL();
+    } catch (err) {
+      throw err;
+    }
+  };
+
+  public getJson = async (file: string) => {
+    try {
```

Comment:
do we need this outer try as well?

Reply:
I think it's better to have it since `getDownloadURL` can throw an error

Reply:
Okay makes sense... but instead of throwing another level up we should handle it here with a console.error or something? Correct me if I'm wrong, but I believe javascript will terminate the program at the point of failure if the error is not handled.

Reply:
For example, if you change `menu.json` to something not valid the whole site will crash. If you handle it here then the site will load and only the menu page will not have images, etc. Unless that's what you want so that we know about it???